src/class/objects.php

```diff
@@ -1518,5 +1530,31 @@ function viewList ($sViewListID = false, $aColsToSkip = array(), $bNoHistory = f
 
         return $nTotal;
     }
+
+    public static function lovd_getObjectLinksHTML($aIDs, $urlFormat)
+    {
+        // Returns a list of gene links in HTML.
+        // Parameter $genes is an array with gene symbols as values.
+        // Parameter $urlPattern designates the target of the links, where %s
+        // will be substituted with the gene symbol.
+
```

Comment:
Please update the function's description; $genes has become $aIDs, and function is not only for genes now.
Please also update $urlPattern to $sURLPattern or $sUrlPattern, whichever you prefer :)